### ISSUE TITLE
HD will not run at all only SD in PI2 

### ISSUE THREAD

**Issue by nb2a** [kodi-pvr/pvr.hdhomerun#18]
I still cannot get any full HD live TV work on a PI2 with any builds at all.
I am out of ideas

Thank you for your work

http://xbmclogs.com/pcevgqxzg

**Comment by zcsizmadia** [comment]
Have you tried the Silicondust python based addon or if you directly connect to your HDHomerun as a Upnp device? Could you play the HD channels from there? 

What hdhomerun tuners are you using?

**Comment by nb2a** [comment]
Thanks for the reply. I am using a HD home run prime that has 3 tuners. The add on does not work either in HD add on if that is what you mean on my PI2 either SD works fine. I have never had any HD issues like this using a windows based system I also have on my network.

**Comment by zcsizmadia** [comment]
What about if you connect to the tuner via the built in hdhomerun vfs or Upnp?

**Comment by zcsizmadia** [comment]
One more thing I forgot to ask. Are the HD channels DRM?

**Comment by nb2a** [comment]
Date: Wed, 24 Feb 2016 04:40:36 -0800

**Comment by nb2a** [comment]
I see in Pseudo TV the channel types seems to be XMLTV if that is what you mean
I am not sure about being DRM honestly

I hope you get this email directly. I did not want to be a pest on your github.

I appreciate your help



Date: Wed, 24 Feb 2016 06:09:53 -0800

**Comment by zcsizmadia** [comment]
Can PseudoTV play it? ARe the channels over the Air or coming from your cable company?

Here is some info about DRM: https://en.wikipedia.org/wiki/Digital_rights_management

Please send me your channel list, it would show if they are DRM or not. 
Go to your browser and go to "http://192.168.1.100/lineup.json", where the IP address is your HDHomerun tuner's IP address.

**Comment by nb2a** [comment]
They are Verizon cable channels and no Pseudo cannot play any HD either Only SD works in either Home Run add on or in Kodi channels itself. This again is not an issue using a windows based setup identical. Only using PI2 setup.

I was looking how to load tar files to maybe try them in PI2 from this link but I cannot figure out how to do it. I am honestly not that familiar with Linux and SSH yet. It was a challenge to simply use SSH to enter MPEG2 license in config.txt. I confirmed that that works too.

http://www.silicondust.com/support/downloads/linux/

Date: Wed, 24 Feb 2016 09:00:26 -0800

**Comment by zcsizmadia** [comment]
I am sorry I cannot help you with that. I really believe your issue is Kodi related (MPEG2 licensing or DRM related) and not related to the pvr addon itself.

**Comment by nb2a** [comment]
I verified the MPEG 2 license code is entered via SSH command and also by OpenElec add on to show license. I have yet to find anyone using this successfully in Pi2 setup.

Thanks for your time

From: nb2a@msn.com
To: reply@reply.github.com
Subject: RE: [pvr.hdhomerun] HD will not run at all only SD in PI2  (#18)
Date: Wed, 24 Feb 2016 12:13:01 -0500




They are Verizon cable channels and no Pseudo cannot play any HD either Only SD works in either Home Run add on or in Kodi channels itself. This again is not an issue using a windows based setup identical. Only using PI2 setup.

I was looking how to load tar files to maybe try them in PI2 from this link but I cannot figure out how to do it. I am honestly not that familiar with Linux and SSH yet. It was a challenge to simply use SSH to enter MPEG2 license in config.txt. I confirmed that that works too.

http://www.silicondust.com/support/downloads/linux/

Date: Wed, 24 Feb 2016 09:00:26 -0800
